### PR TITLE
Create suzumi.badboy.json

### PR DESCRIPTION
**Pull request by s3mant**

### CODE REVIEW

File: api.badboy.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Bad Boy's Api",
+  "repo": "https://github.com/Bad-Boy-Codes/bad-boy-codes.github.io",
+  "owner": {
+    "username": "badboy",
```
Comment (MaskDuck) [inline api.badboy.json:5]: username is wrong ig
```suggestion
    "username": "Bad-Boy-Codes",
```

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Comment by MaskDuck** [comment]
and also it must be placed in the domains folder

**Comment by Bad-Boy-Codes** [comment]
> and also it must be placed in the domains folder

oh lol okay xd

**Comment by MaskDuck** [comment]
idk

**Review by s3mant — COMMENTED** [review]
a

**Review by MaskDuck — DISMISSED** [review]
not sus

**Review by mtgsquad — APPROVED** [review]

**Pull request merged**